spec/13-functions.md

```diff
@@ -74,6 +74,7 @@ A function is called via the function-call operator [`()`](10-expressions.md#fun
 
   <i>return-type:</i>
     : <i>type-declaration</i>
+    : void
```

Comment:
No I think it's fine - void is valid only as return type, not any type declaration.